helm/korifi/kpack-image-builder/manifests.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: korifi-kpack-image-builder-mutating-webhook-configuration
+  annotations:
+    cert-manager.io/inject-ca-from: '{{ .Release.Namespace }}/korifi-controllers-serving-cert'
+webhooks:
+  - admissionReviewVersions:
+      - v1
+      - v1beta1
+    clientConfig:
+      service:
+        name: korifi-controllers-webhook-service
+        namespace: '{{ .Release.Namespace }}'
+        path: /mutate-korifi-cloudfoundry-org-v1alpha1-kpack-image-builder-finalizer
+    failurePolicy: Fail
+    name: mcf-kib-finalizer.korifi.cloudfoundry.org
+    rules:
+      - apiGroups:
+          - korifi.cloudfoundry.org
```

Comment:
It's a bit weird how these resources have been mixed here. It looks like we could be operating on korifi.cloudfoundry.org/build and kpack.io/buildworkload, even though those don't exist.

I guess it's just a kubebuilder generation thing, and not massively important.